src/vuestic-theme/vuestic-components/vuestic-simple-select/VuesticSimpleSelect.vue

```diff
@@ -57,48 +61,58 @@
       name: {
         type: String,
         default: 'simple-select'
+      },
+    },
+    data () {
+      return {
+        validated: false,
+        displayValue: this.value,
       }
     },
-    mounted () {
-      this.updateDisplayValue(this.value)
-      this.$emit('input', this.value)
+    watch: {
+      value: {
+        handler (value) {
+          this.displayValue = value || ''
+        },
+        immediate: true,
+      }
     },
-
-    updated: function () {
-      this.updateDisplayValue(this.value)
+    computed: {
+      filteredList () {
+        return this.options.filter(item => item.search(this.displayValue) === 0)
```

Comment:
@asvae: What about the simple-select where options are provided as a list of dictionaries instead of a list and option-key is prop is passed. In that case this line is throwing error. 
```
        [
          {
            id: 1,
            description: 'First option',
          },
          {
            id: 2,
            description: 'Second option',
          },
          {
            id: 3,
            description: 'Third option',
          },
        ]
```

Reply:
fixed.